subscription_oca/models/sale_order.py

```diff
@@ -58,13 +58,16 @@ def create_subscription(self, lines, subscription_tmpl):
                 }
             )
             rec.action_start_subscription()
-            self.subscription_ids = [(4, rec.id)]
+            self.write({"subscription_ids": [Command.link(rec.id)]})
```

Comment:
I'm wondering why this is necessary as `sale_order_id' fields is filled in above ?

Reply:
Can't see a reason either (it's been there from the beginning on v15). Of what I can see it works without. I remove it.

Reply:
Yes, because it generated unnecessary write()